spot-ml/src/main/scala/org/apache/spot/SpotLDAWrapper.scala

```diff
@@ -0,0 +1,189 @@
+package org.apache.spot
```

Comment:
I really like more having SpotLDA* code all together in its own package or folder and just leave Suspicious* code in the _root_.